src/scripts/content/kantree.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+/* Features: */
+/* - Add timer to the card */
+/* - Add timer to the sub-tasks list */
+/* - Get toggl project name from kantree card */
+/* - Get toggl tags from kantree card */
+/* - Handle card view mode changes */
+console.log('Toggl Button loaded for kantree.');
```

Comment:
Was it commited by an accident? 🙈

Reply:
I don't think so, I think it's quite useful to see if integration was loaded, I was inspired by https://github.com/toggl/toggl-button/blob/6565ec50c6e4c667e81ba9fdccf53a21c55dc925/src/scripts/content/kanboard.js#L3
Don't you?